docs/fsharp/tutorials/asynchronous-and-concurrent-programming/async.md

```diff
@@ -25,7 +25,7 @@ The key concept to understand is that an async expression’s type is `Async<'T>
 
 For example, say you wanted to download the HTML from dotnetfoundation.org without blocking the main thread. You can accomplish it like this:
 
-```fs
+```fsharpharp
```

Comment:
should be fs only. maybe search your working branch to see if you have other instances of fsharpharp or csharpharp